static/app/views/dashboardsV2/widgetBuilder/widgetBuilder.spec.tsx

```diff
@@ -495,10 +497,16 @@ describe('WidgetBuilder', function () {
         dashboard: testDashboard,
       });
 
-      userEvent.type(
-        await screen.findByRole('textbox', {name: 'Search events'}),
-        'color:blue{enter}'
-      );
+      const search = await screen.findByTestId(/smart-search-input/);
+
+      userEvent.click(search);
+
+      // Use fireEvent for performance reasons as this test suite is slow
+      fireEvent.paste(search, {
+        target: {value: 'color:blue'},
+        clipboardData: {getData: () => 'color:blue'},
+      });
+      fireEvent.keyPress(search, {key: 'Enter', code: 13, charCode: 13});
```

Comment:
```suggestion
      userEvent..keyboard(...)
```
🤷‍♀️

Reply:
Seems like very little impact on my local run - I'll swap this to use userEvent.keyboard